security/zynqmp/bifs/keys.bif

```diff
@@ -0,0 +1,7 @@
+the_ROM_image:
+{
+	[ppkfile] <keys/PPK.pem>
+	[pskfile] <keys/PSK.pem>
+	[spkfile] <keys/SPK.pem>
+	[sskfile] <keys/SSK.pem>
+}
```

Comment:
[nit] May consider fixing the last line on this file to avoid the odd commit change later.

Reply:
ok